packages/react/src/components/listBox/ListBox.tsx

```diff
@@ -9,14 +9,39 @@ import {ItemBoxLoading} from '../loading/components/ItemBoxLoading';
 export type IItemBoxPropsWithIndex = {index?: number} & IItemBoxProps;
 
 export interface IListBoxOwnProps {
+    /**
+     * The content shown in the list of items when there are no items to display.
+     *
+     * @default {value: 'No Items'}
+     */
     noResultItem?: IItemBoxProps;
+    /**
+     * Additional CSS classes to set on the list element
+     */
     classes?: string[];
     id?: string;
     multi?: boolean;

```

Comment:
These two not the right to JSDoc?

Reply:
I only added jsdocs to props that are visible in the props table